### PR TITLE
Add Seun's welcome blog post

### PR DESCRIPTION
**Pull request by NPDebs**
**This PR fixes issue #849.
A welcome blog post has been added for @seunolufemi123!** 🎉 

See screenshot below:
 
Screenshot (748)

## FOR CONTRIBUTOR

* [x] I have read the [CONTRIBUTING.md](https://github.com/open-life-science/open-life-science.github.io/blob/main/CONTRIBUTING.md) document

PR Type: 
* [x] This PR adds a new blog post
* [ ] This PR does something else (explain above)

 
## FOR REVIEWERS

Thanks for taking the time to review! :heart:

Here are the list of things to make sure of:
* [x] The website builds (a check will fail if not)
* [x] All images have been added within the Pull Request and they have Alt text
* [x] If there are paragraphs or text, the key messages are highlighted
* [x] All internal links (within OLS website) use the [`{% link path_to_file.md %}` format](https://jekyllrb.com/docs/liquid/tags/#link)
* [x] The preview corresponds to the changes described in the Pull Request
* [x] The code is tidy and passes the linting tests

### CODE REVIEW

File: _posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+We are grateful to all our applicants who presented strong project plans and shared their vision for this role. Since we could only offer one position, the decision was based on the applicant's project goals and the feasibility of the OLS team to support them over the next eight months as part of the Open Seeds OLS-9 delivery plans.
+
+**We are delighted to announce that Seun has been offered this role of the OLS Resident Fellow!**
+Seun will work with Malvika Sharan and Tajuddeen (Taj) Gwadabe as part of the OLS-9 delivery team.
+
+<div class="columns">
+  <div class="column is-4" markdown="1">
+  ![A photo of Seun, a man from Nigeria. He is wearing a smile and a colourful, bright African round-neck shirt.](https://hackmd.io/_uploads/B1EogG_uA.jpg)
+  [add links to different orgs above]
```
Comment (NPDebs) [inline _posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md:24]: @malvikasharan Are you happy with the links you've added? 😀

I'd like to delete your reminder before we go live.

File: _posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Welcoming OLS Resident Fellow Seun Olufemi to OLS-9 Delivery Team
+authors:
+- malvikasharan
+- seunolufemi123
+image: https://images.unsplash.com/photo-1554457606-ed16c39db884
```
Comment (NPDebs) [inline _posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md:7]: @seunolufemi123 Are you happy with this image or would you prefer [this](https://unsplash.com/photos/white-ceramic-teapot-on-white-wooden-box-OGND72jS-HE) or something else?

File: _posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  </div>
+  <div class="column" markdown="1">
+
+## A Biochemistry Researcher and Champion of Open Practices
+
+Seun, a biochemistry graduate from [Ladoke Akintola University of Technology](https://www.lautech.edu.ng/), is passionate about understanding the molecular pathogenesis of disease and developing therapeutic against it.
+He is also a strong champion of digital skills and open science.
+He is a Graduate Research Intern at the [Institute of Human Virology](https://ihvnigeria.org/), Nigeria, studying the molecular mechanism of disease progression.
+He is also a Research and Teaching Assistant at [Helix Biogen Institute](https://www.helixbiogeninstitute.org/), where he supports translational research and training in bioinformatics and vaccine design.
+Although an Early Career Researcher himself, Seun has published eight articles in peer-reviewed journals.
```
Comment (malvikasharan) [inline _posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md:34]:
```suggestion
Although an Early Career Researcher himself, Seun has published eight  peer-reviewed articles.
```

File: _posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md (hunk 1 above)
Comment (malvikasharan) [reply _posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md:24]: Looks great! Thank you Debs.

**Comment by yochannah** [comment]
quick note - are we still adding Pidgin as well as English?

**Comment by yochannah** [comment]
quick note - are we still adding Pidgin as well as English? tagging @seunolufemi123 @NPDebs @Tajuddeen1

**Comment by NPDebs** [comment]
> quick note - are we still adding Pidgin as well as English? tagging @seunolufemi123 @NPDebs @Tajuddeen1 

I asked Debs the same question while creating this PR. She thought it might make the already-lengthy post a drag to read.

Eager to hear other opinions. 💡

**Comment by Tajuddeen1** [comment]
Does @seunolufemi123 has an opinion?

**Comment by seunolufemi123** [comment]
@Tajuddeen1 @NPDebs @yochannah, I am quite indifferent, as much as I would love it to be in Pidgin. I am also of the opinion that the blog post shouldn't be too long unless we want to have it in another blog post

File: _posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md (hunk 1 above)
Comment (bebatut) [reply _posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md:24]: Where are the links?

Comment (NPDebs) [reply _posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md:24]: I believe the links are the ones below 👇🏽

Meaning, we can remove this line.

Comment (malvikasharan) [reply _posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md]: yes, please delete that - we needed to add link to the OLS-9 Blog.

Comment (NPDebs) [reply _posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md]: Deleted already. 😃

**Comment by yochannah** [comment]
Let's do a second post in pidgin, then. I'll create an issue for it now.

**Review by yochannah — APPROVED** [review]

**Pull request merged**